modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceOrderDisplayPageTemplates.testcase

```diff
@@ -332,6 +332,97 @@ definition {
 		}
 	}
 
+	@description = "This is a test to check the COMMERCE-12060. I want to be able to use the Order Item fragment for displays correctly the options for the product bundles"
```

Comment:
As an admin I want the Order Item fragment to display correctly the options for the product bundles